app-text/cutemarked/cutemarked-0.11.3.recipe

```diff
@@ -0,0 +1,69 @@
+SUMMARY="Qt Markdown editor"
+DESCRIPTION="A Qt-based, free and open source Markdown editor with live HTML preview, math expressions, code and markdown syntax highlighting."
+HOMEPAGE="https://cloose.github.io/CuteMarkEd"
+COPYRIGHT="2013 Christian Loose"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/cloose/CuteMarkEd/archive/v$portVersion.zip"
+CHECKSUM_SHA256="a8f1c8f22ce52ff8a878607112a44ca3d45015120d07b301f78fc9631b894775"
+SOURCE_DIR="CuteMarkEd-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+		cutemarked$secondaryArchSuffix = $portVersion
+		app:CuteMarkEd$secondaryArchSuffix = $portVersion
+		"
+		
+REQUIRES="
+		haiku$secondaryArchSuffix
+		hunspell$secondaryArchSuffix
+		lib:libmarkdown$secondaryArchSuffix
+		lib:libQt5WebKit$secondaryArchSuffix
+		lib:libQt5WebKitWidgets$secondaryArchSuffix
+		lib:libQt5Core$secondaryArchSuffix
+		lib:libQt5Gui$secondaryArchSuffix 
+		lib:libQt5Widgets$secondaryArchSuffix
+		lib:libQt5Network$secondaryArchSuffix
+		lib:libQt5PrintSupport$secondaryArchSuffix 
+		lib:libQt5Test$secondaryArchSuffix
+		"
+
+BUILD_REQUIRES="
+		haiku${secondaryArchSuffix}_devel
+		hunspell${secondaryArchSuffix}_devel
+		devel:libmarkdown$secondaryArchSuffix 
+		devel:libQt5WebKit$secondaryArchSuffix
+		devel:libQt5WebKitWidgets$secondaryArchSuffix
+		devel:libQt5Core$secondaryArchSuffix 
+		devel:libQt5Gui$secondaryArchSuffix
+		devel:libQt5Widgets$secondaryArchSuffix
+		devel:libQt5Network$secondaryArchSuffix
+		devel:libQt5QmlDebug$secondaryArchSuffix
+		devel:libQt5PrintSupport$secondaryArchSuffix
+		devel:libQt5Test$secondaryArchSuffix			
+		"
+
```

Comment:
Our [guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines) require not having a newline here